web-services/query/src/main/java/datawave/webservice/query/util/LookupUUIDUtil.java

```diff
@@ -418,7 +419,7 @@ public <T> T createUUIDQueryAndNext(final AbstractUUIDLookupCriteria unvalidated
         return response;
     }
     
-    private String getAuths(String logicName, String queryAuths, Principal principal) {
+    private String getAuths(String logicName, MultivaluedMap<String,String> queryParameters, String queryAuths, Principal principal) {
```

Comment:
You are correct, I was unable to figure out how to create the Query object easily from the queryParameters to modify this section.  Let me take a look to see if I can do that.  Otherwise I will remove it.